### PR TITLE
zh: add what's new in v1.2

### PR DESCRIPTION
**Pull request by DanielZhangQD**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/whats-new-in-v1.2.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: What's New in TiDB Operator 1.2
+---
+
+# What's New in TiDB Operator 1.2
+
+TiDB Operator 1.2 在 1.1 基础上完善了 TidbCluster CR（Custom Resource）的功能，支持对 Pump 的完整生命周期管理，并对 TidbMonitor controller 进行了重构，支持监控多个集群、Remotewrite、Thanos sidecar 等多个新功能，并针对 TiDB Operator 部署做了优化，支持部署 namespace scoped 以及支持部署多套 TiDB Operator 分别管理不同的 TiDB 集群。同时在易用性上做了许多改进，支持通过 CR 管理 DM 2.0 集群，为 TiKV、TiFlash 自定义 Store 标签等等。以下是主要变化：
```
Comment (csuzhangxc) [inline zh/whats-new-in-v1.2.md:7]: 这一段似乎略有一点不易抓住重点  @qiancai 有啥建议没有

**Review by csuzhangxc — APPROVED** [review]

File: zh/whats-new-in-v1.2.md (hunk 1 above)
Comment (DanielZhangQD) [reply zh/whats-new-in-v1.2.md:7]: 是，等caiqian 看看吧

Comment (qiancai) [reply zh/whats-new-in-v1.2.md:7]: 这里提到的功能都在下面的列表中也提了是吗？如果是，感觉不写也可以的，除非特别想强调下。

Comment (qiancai) [reply zh/whats-new-in-v1.2.md:7]: 也可以按照这个思路来 “TiDB Operator 1.2 引入了以下关键特性，从扩展性、易用性、安全性方面帮助你更轻松便捷地管理 TiDB 集群。”

Comment (DanielZhangQD) [reply zh/whats-new-in-v1.2.md:7]:
```suggestion
TiDB Operator 1.2 引入了以下关键特性，从扩展性、易用性、安全性等方面帮助你更轻松地管理 TiDB 集群及其周边工具。
```

File: zh/whats-new-in-v1.2.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## 安全性
+
+- [TiDB Lightning chart 和 TiKV Importer chart 支持配置 TLS](enable-tls-between-components.md)
+
+## 实验性特性
+
+- [跨多个 Kubernetes 集群部署一个 TiDB 集群](deploy-tidb-cluster-across-multiple-kubernetes.md)
+
+TiDB Operator 在 Kubernetes 上部署参见[安装文档](deploy-tidb-operator.md)，CRD 文档参见 [API References](https://github.com/pingcap/tidb-operator/blob/master/docs/api-references/docs.md)。
```
Comment (qiancai) [inline zh/whats-new-in-v1.2.md:36]: 这句话为啥要写到 “跨多个 Kubernetes 集群部署一个 TiDB 集群” 后面呢，是在不说过程中需要用到 deploy-tidb-operator.md 里的内容吗？

File: zh/whats-new-in-v1.2.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+## 扩展性
+
+- 支持为 TidbCluster 中的 Pod 与 service 自定义 label 及 annotation
+- 支持为所有 TiDB 组件设置 podSecurityContext、topologySpreadConstraints
+- 支持对 Pump 的完整生命周期管理
+- TidbMonitor 支持[监控多个 TidbCluster](monitor-a-tidb-cluster.md#多集群监控)
+- TidbMonitor 支持 remotewrite
+- TidbMonitor 支持[配置 Thanos sidecar](aggregate-multiple-cluster-monitor-data.md)
+- TidbMonitor 管理资源从 Deployment 变为 StatefulSet
```
Comment (qiancai) [inline zh/whats-new-in-v1.2.md:17]: 这条需要写下这么做的好处吗

File: zh/whats-new-in-v1.2.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+# What's New in TiDB Operator 1.2
+
+TiDB Operator 1.2 引入了以下关键特性，从扩展性、易用性、安全性等方面帮助你更轻松地管理 TiDB 集群及其周边工具。
+
+## 扩展性
+
+- 支持为 TidbCluster 中的 Pod 与 service 自定义 label 及 annotation
+- 支持为所有 TiDB 组件设置 podSecurityContext、topologySpreadConstraints
+- 支持对 Pump 的完整生命周期管理
```
Comment (qiancai) [inline zh/whats-new-in-v1.2.md:13]: Pump 要不给个链接 https://docs.pingcap.com/zh/tidb/stable/tidb-binlog-overview#pump

Comment (DanielZhangQD) [reply zh/whats-new-in-v1.2.md:13]:
```suggestion
- 支持对 [Pump](https://docs.pingcap.com/zh/tidb/stable/tidb-binlog-overview#pump) 的完整生命周期管理
```

File: zh/whats-new-in-v1.2.md (hunk 3 above)
Comment (DanielZhangQD) [reply zh/whats-new-in-v1.2.md:17]:
```suggestion
- TidbMonitor 管理资源从 Deployment 变为 StatefulSet，以支持更多灵活性
```

File: zh/whats-new-in-v1.2.md (hunk 2 above)
Comment (DanielZhangQD) [reply zh/whats-new-in-v1.2.md:36]: 这句是文档最后的一句，我删掉吧

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**